core/src/main/java/io/questdb/cairo/TableNameRegistryFileStore.java

```diff
@@ -132,23 +132,22 @@ private void compactTableNameFile(Map<CharSequence, TableToken> nameTableTokenMa
             Path path2 = Path.getThreadLocal2(configuration.getRoot())
                     .concat(TABLE_REGISTRY_NAME_FILE).put('.').put(lastFileVersion + 1).$();
             if (ff.rename(path, path2) == Files.FILES_RENAME_OK) {
-                LOG.info().$("compacted tables file [path=").$(path2).$(']').$();
+                LOG.info().$("compacted tables file [path=").utf8(path2).I$();
                 lastFileVersion++;
                 currentOffset = newAppendOffset;
                 // best effort to remove old files, but we don't care if it fails
                 path.trimTo(pathRootLen).concat(TABLE_REGISTRY_NAME_FILE).put('.').put(lastFileVersion - 1).$();
                 ff.remove(path);
             } else {
                 // Not critical, if rename fails, compaction will be done next time
-                LOG.error().$("could not rename tables file, tables file will not be compacted [from=").$(path)
-                        .$(", to=").$(path2).$(']').$();
+                LOG.error().$("could not rename tables file, tables file will not be compacted [from=").utf8(path)
+                        .$(", to=").utf8(path2).I$();
             }
         } finally {
             path.trimTo(pathRootLen).concat(TABLE_REGISTRY_NAME_FILE).put('.').put(lastFileVersion).$();
             tableNameMemory.smallFile(ff, path, MemoryTag.MMAP_DEFAULT);
             tableNameMemory.jumpTo(currentOffset);
         }
-        tableNameMemory.jumpTo(currentOffset);
```

Comment:
nit: don't you like double jumps? 😜

Reply:
yeah!

Reply:
are they a thing?, I mean, no, no?, unless there is some magic I do not perceive, like this makes the memory manager to flip a bit clearing buffers or something like that?, at this point I am not sure of anything :)

Reply:
This call was redundant, no worries.